### PR TITLE
Updates for the 2.0 release

### PR DESCRIPTION
**Pull request by lilleyse**
Updated `CHANGES.md` and `README.md` in preparation for releasing the 2.0 version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +122,6 @@ processGltf(gltf, options)
 |`--json`, `-j`|Convert the input glb to glTF.|No, default `false`|
 |`--separate`, `-s`|Write separate buffers, shaders, and textures instead of embedding them in the glTF.|No, default `false`|
 |`--separateTextures`, `-t`|Write out separate textures only.|No, default `false`|
-|`--checkTransparency`|Do a more exhaustive check for texture transparency by looking at the alpha channel of each pixel. By default textures are considered to be opaque.|No, default `false`|
```
[the remark this answers is not in the document]
Comment (ggetz) [reply README.md]: Also the examples are out of date:
```
./specs/data/box.gltf
```
Should be
```
./specs/data/1.0/box-textured-X/box-textured-X.gltf
```

File: README.md [hunk 2]
```diff
@@ -106,7 +122,6 @@ processGltf(gltf, options)
...
 |`--stats`|Print statistics to console for input and output glTF files.|No, default `false`|
```
Comment (ggetz) [inline README.md]: When running `--stats`, the reported input `Images`, `Nodes`, `Meshes`, `Materials`, and `Animations` are all undefined. Output stats are fine.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lilleyse) [reply README.md]: I changes the examples to just be generic `model.gltf` rather than an actual file.

File: README.md (hunk 2 above)
Comment (lilleyse) [reply README.md]: It was failing for glTF 1.0 models.

I changed the code and doc to only print stats once for the final model.

**Comment by lilleyse** [comment]
Thanks @ggetz, updated.

**Comment by ggetz** [comment]
Thanks @lilleyse !

**Pull request merged**